### ISSUE TITLE
Can‘t download the supernet

### ISSUE THREAD

**Issue by HardyC-1021** [mikelzc1990/nsganetv2#17]
I want to download the supernet 'ofa_mbv3_d234_e346_k357_w1.0 ', but this link is Forbidden.
Download the pre-trained (on ImageNet) supernet from [here](https://hanlab.mit.edu/files/OnceForAll/ofa_nets/). 
Where could I get it again?  Thanks Thanks Thanks!

**Comment by xmuhlw** [comment]
U can get this file in link 'https://drive.google.com/drive/folders/10leLmIiMtaRu4J46KwrBaMydvQt0qFuI'.
That is from 'Once for all‘ repository, https://github.com/mit-han-lab/once-for-all.

**Comment by HardyC-1021** [comment]
Thank you very much !

**Comment by catinthedawn** [comment]
Can you send me a copy of Supernet? Both links are Forbidden. thanks!

**Comment by xmuhlw** [comment]
> Can you send me a copy of Supernet? Both links are Forbidden. thanks!

Of course, this is a copy of the supernet 'ofa_mbv3_d234_e346_k357_w1.0 , I upload it to google drive 'https://drive.google.com/file/d/1rtG6lzp1bcT5MEm1VxeXoTutYq_skAxx/view?usp=drive_link';

**Comment by catinthedawn** [comment]
> > Can you send me a copy of Supernet? Both links are Forbidden. thanks!
> 
> Of course, this is a copy of the supernet 'ofa_mbv3_d234_e346_k357_w1.0 , I upload it to google drive 'https://drive.google.com/file/d/1rtG6lzp1bcT5MEm1VxeXoTutYq_skAxx/view?usp=drive_link';

thank you very much!But when I clicked on the link it said I don't have permission. I have applied for access permission, could you please go through it?

**Comment by xmuhlw** [comment]
> > > Can you send me a copy of Supernet? Both links are Forbidden. thanks!
> > 
> > 
> > Of course, this is a copy of the supernet 'ofa_mbv3_d234_e346_k357_w1.0 , I upload it to google drive 'https://drive.google.com/file/d/1rtG6lzp1bcT5MEm1VxeXoTutYq_skAxx/view?usp=drive_link';
> 
> thank you very much!But when I clicked on the link it said I don't have permission. I have applied for access permission, could you please go through it?

Now please check ur permission, i have just gone through it.

**Comment by catinthedawn** [comment]
> > > > Can you send me a copy of Supernet? Both links are Forbidden. thanks!
> > > 
> > > 
> > > Of course, this is a copy of the supernet 'ofa_mbv3_d234_e346_k357_w1.0 , I upload it to google drive 'https://drive.google.com/file/d/1rtG6lzp1bcT5MEm1VxeXoTutYq_skAxx/view?usp=drive_link';
> > 
> > 
> > thank you very much!But when I clicked on the link it said I don't have permission. I have applied for access permission, could you please go through it?
> 
> Now please check ur permission, i have just gone through it.

Thank you very much!!

**Comment by GREATLYJDN** [comment]
Can you send me a copy of Supernet? Both links are Forbidden. thanks! @xmuhlw

**Comment by GREATLYJDN** [comment]
Can you send me a copy of Supernet? Both links are Forbidden. thanks! @catinthedawn

**Comment by xmuhlw** [comment]
> Can you send me a copy of Supernet? Both links are Forbidden. thanks! @xmuhlw

Of course,  I set the download permission so that everyone can access it.

**Comment by GREATLYJDN** [comment]
> Thank you very much !

**Comment by GREATLYJDN** [comment]
Thank you very much ! @xmuhlw @catinthedawn
